DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/PointToSpeckleConverter.cs

```diff
@@ -0,0 +1,15 @@
+using ArcGIS.Core.Geometry;
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Models;
+using Objects.Geometry;
+using Speckle.Converters.Common;
+
+namespace Speckle.Converters.ArcGIS3.Geometry;
+
+[NameAndRankValue(nameof(MapPoint), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
+internal class PointToSpeckleConverter : IHostObjectToSpeckleConversion, IRawConversion<MapPoint, Point>
+{
+  public Base Convert(object target) => RawConvert((MapPoint)target);
+
+  public Point RawConvert(MapPoint target) => new(target.X, target.Y, target.Z, "m");
```

Comment:
I'm assuming this is still pending having the units passed in as part of the context object.
We should add a POC comment here at least.